incubator/binding-pgsql/src/main/java/io/aklivity/zilla/runtime/binding/pgsql/parser/listener/SqlCreateStreamListener.java

```diff
@@ -36,9 +36,9 @@ public SqlCreateStreamListener(
         this.tokens = tokens;
     }
 
-    public StreamInfo streamInfo()
+    public Stream streamInfo()
```

Comment:
If you want to have a suffix for these model class names, consider `Model` instead of `Info`.
Alternatively consider a prefix like `PgsqlStream`.

Reply:
I actually removed suffixes from all other places this got left over